### ISSUE TITLE
 ValueError: Unknown Openhab type: Call 

### ISSUE THREAD

**Issue by DerOetzi** [spacemanspiff2007/HABApp#258]
Please add Call item type for telephone calls.

Thank you

**Comment by spacemanspiff2007** [comment]
According to the [docs](https://www.openhab.org/docs/concepts/items.html#items) this item type does not exist.
What does it do?

**Comment by DerOetzi** [comment]
That is really interesting the type is used at least by the avmfritz  binding for information about 
 incoming and outgoing phone calls. See: https://www.openhab.org/addons/bindings/avmfritz/#supported-channels 

If I got it correctly from the java file it holds a value of type StringListType:

https://github.com/openhab/openhab-core/blob/79edf2b9e643ad802c31b2e5f54edb93534277fd/bundles/org.openhab.core/src/main/java/org/openhab/core/library/items/CallItem.java

**Comment by spacemanspiff2007** [comment]
Could you post the event that generates the error and the value that you get when you request the item in the API explorer?
Make sure to redact your telephone numbers!

**Comment by DerOetzi** [comment]
The error occures directly on startup:

```
[2022-01-06 12:07:29,823] [HABApp.openhab.connection]     INFO | Connected to OpenHAB instance b01c3f12-c476-4c2c-9432-cb3ab7df8109
[2022-01-06 12:07:29,823] [HABApp.openhab.connection]     INFO | OpenHAB version 3.2.0 (Release Build)
[2022-01-06 12:07:30,350] [           HABApp.openhab]    ERROR | Error Unknown Openhab type: Call for incomingFritzbox in map_items:
[2022-01-06 12:07:30,350] [           HABApp.openhab]    ERROR | File "/usr/local/lib/python3.8/site-packages/HABApp/openhab/map_items.py", line 109, in map_item
[2022-01-06 12:07:30,350] [           HABApp.openhab]    ERROR |     17   def map_item(name: str, type: str, value: Optional[str],
[2022-01-06 12:07:30,350] [           HABApp.openhab]    ERROR |     18                tags: FrozenSet[str], groups: FrozenSet[str], metadata: Optional[Dict[str, Dict[str, Any]]]) -> Optional['HABApp.openhab.items.OpenhabItem']:
[2022-01-06 12:07:30,350] [           HABApp.openhab]    ERROR |  (...)
[2022-01-06 12:07:30,350] [           HABApp.openhab]    ERROR |     105  
[2022-01-06 12:07:30,351] [           HABApp.openhab]    ERROR |     106          if type == "Player":
[2022-01-06 12:07:30,351] [           HABApp.openhab]    ERROR |     107              return PlayerItem(name, value, tags=tags, groups=groups, metadata=meta)
[2022-01-06 12:07:30,351] [           HABApp.openhab]    ERROR |     108  
[2022-01-06 12:07:30,351] [           HABApp.openhab]    ERROR | --> 109          raise ValueError(f'Unknown Openhab type: {type} for {name}')
[2022-01-06 12:07:30,351] [           HABApp.openhab]    ERROR |     110  
[2022-01-06 12:07:30,351] [           HABApp.openhab]    ERROR |     ..................................................
[2022-01-06 12:07:30,351] [           HABApp.openhab]    ERROR |      name = 'incomingFritzbox'
[2022-01-06 12:07:30,351] [           HABApp.openhab]    ERROR |      type = 'Call'
[2022-01-06 12:07:30,351] [           HABApp.openhab]    ERROR |      value = None
[2022-01-06 12:07:30,351] [           HABApp.openhab]    ERROR |      Optional = typing.Optional
[2022-01-06 12:07:30,351] [           HABApp.openhab]    ERROR |      tags = frozenset({'Status'})
[2022-01-06 12:07:30,351] [           HABApp.openhab]    ERROR |      FrozenSet = typing.FrozenSet
[2022-01-06 12:07:30,351] [           HABApp.openhab]    ERROR |      groups = frozenset({'callmonitorFritzbox'})
[2022-01-06 12:07:30,351] [           HABApp.openhab]    ERROR |      metadata = {'channel': {'value': 'avmfritz:fritzbox:10_0_10_1:incoming_call'},
[2022-01-06 12:07:30,351] [           HABApp.openhab]    ERROR |                  'semantics': {'value': 'Point_Status',
[2022-01-06 12:07:30,351] [           HABApp.openhab]    ERROR |                                'config': {...}}}
[2022-01-06 12:07:30,352] [           HABApp.openhab]    ERROR |      Dict = typing.Dict
[2022-01-06 12:07:30,352] [           HABApp.openhab]    ERROR |      Any = typing.Any
[2022-01-06 12:07:30,352] [           HABApp.openhab]    ERROR |      PlayerItem = <class 'HABApp.openhab.items.string_item.PlayerItem'>
[2022-01-06 12:07:30,352] [           HABApp.openhab]    ERROR |      meta = immutables.Map({'semantics': MetaData(value='Point_Status', config=immutables.Map({'isPointOf': 'callmonitorFritzbox'})), 'channel': MetaData(value='avmfritz:fritzbox:10_0_10_1:incoming_call', config=immutables.Map({}))})
[2022-01-06 12:07:30,352] [           HABApp.openhab]    ERROR |     ..................................................
[2022-01-06 12:07:30,352] [           HABApp.openhab]    ERROR | 
[2022-01-06 12:07:30,352] [           HABApp.openhab]    ERROR | ValueError: Unknown Openhab type: Call for incomingFritzbox
```
It has no value at this moment.

On incoming call the rest {{endpoint}}/rest/items/:itemname/state for this item returns:

01234,056789

while 01234 is the target number and 056789 is the calling number

**Comment by spacemanspiff2007** [comment]
> On incoming call the rest {{endpoint}}/rest/items/:itemname/state for this item returns:

Could you post the complete response with redacted numbers, please?

**Comment by DerOetzi** [comment]
What do you mean with complete response? This is the complete state response!

**Comment by DerOetzi** [comment]
Hope this is the request you expect:

```json
{
    "link": "http://10.0.10.200:8080/rest/items/incomingFritzbox",
    "state": "01234,056789",
    "stateDescription": {
        "pattern": "%1$s to %2$s",
        "readOnly": true,
        "options": []
    },
    "editable": false,
    "type": "Call",
    "name": "incomingFritzbox",
    "label": "Eingehender Anruf",
    "tags": [
        "Status"
    ],
    "groupNames": [
        "callmonitorFritzbox"
    ]
}
```

**Comment by spacemanspiff2007** [comment]
I've added support for the CallItem in the dev branch, however the event definition is still missing.
Could you test the dev branch and post the error when the item gets an update? Thanks

**Comment by DerOetzi** [comment]
Seems to work well. There isn't an error on startup anymore and event seems to work as well:

```logging
[2022-01-07 16:03:01,133] [          HABApp.EventBus]     INFO |     incomingFritzbox: <ItemStateChangedEvent name: incomingFritzbox, value: 01234,056789, old_value: None>
```

**Comment by spacemanspiff2007** [comment]
Shouldn't it be split into two entries? 01234 and 056789?

**Comment by DerOetzi** [comment]
I would suggest to always split the value even if there is no comma inside to enforce that the CallItem type always holds a tuple as value type. 

So remove the second condition [here](https://github.com/spacemanspiff2007/HABApp/blob/e4a05127f5bc78e6b2499e0f655691f73617e009/src/HABApp/openhab/items/string_item.py#L20)

Like this:

```python
        if isinstance(new_value, str):
            new_value = tuple(new_value.split(','))
```

**Comment by spacemanspiff2007** [comment]
That's a good idea! I'll modify it accordingly.

**Comment by spacemanspiff2007** [comment]
Included with 1.0
